### PR TITLE
[thermalctld] Optimize the thermal policy loop to make it execute every 60 seconds

### PR DESCRIPTION
**Pull request by Junchao-Mellanox**
Why I did this?
In regression, we found that thermal policy loop takes 8 to15 seconds sometimes which would enhance  the loop interval to 68 to 75 seconds. This change is to make the loop interval more accurate.

How I did this?
Record the elapse time for thermal policy, and start next iteration in (60 - elapse) seconds.

How I test this?
Manual test on Mellanox 2410.

### CODE REVIEW

File: sonic-thermalctld/scripts/thermalctld [hunk 1]
```diff
@@ -638,12 +641,23 @@ class ThermalControlDaemon(DaemonBase):
...
-        while not self.stop_event.wait(ThermalControlDaemon.INTERVAL):
+        wait_time = ThermalControlDaemon.INTERVAL
+        while not self.stop_event.wait(wait_time):
+            begin = time.time()
             try:
                 if thermal_manager:
                     thermal_manager.run_policy(chassis)
             except Exception as e:
                 logger.log_error('Caught exception while running thermal policy - {}'.format(e))
+            elapse = time.time() - begin
```
Comment (jleveque) [inline sonic-thermalctld/scripts/thermalctld:652]: Rename variable `elapsed`

File: sonic-thermalctld/scripts/thermalctld [hunk 2]
```diff
@@ -638,12 +641,23 @@ class ThermalControlDaemon(DaemonBase):
...
+            if elapse < ThermalControlDaemon.INTERVAL:
+                wait_time = ThermalControlDaemon.INTERVAL - elapse
+            else:
+                wait_time = ThermalControlDaemon.FAST_START_INTERVAL
+
+            if elapse > ThermalControlDaemon.RUN_POLICY_ELAPSE_THRESHOLD:
+                logger.log_warning('Run thermal policy takes {} seconds, '
```
Comment (jleveque) [inline sonic-thermalctld/scripts/thermalctld:659]: s/Run thermal policy/Thermal policy execution/

File: sonic-thermalctld/scripts/thermalctld (hunk 1 above)
Comment (Junchao-Mellanox) [reply sonic-thermalctld/scripts/thermalctld]: Fixed.

File: sonic-thermalctld/scripts/thermalctld (hunk 2 above)
Comment (Junchao-Mellanox) [reply sonic-thermalctld/scripts/thermalctld]: Fixed。

File: sonic-thermalctld/scripts/thermalctld [hunk 3]
```diff
@@ -607,6 +607,9 @@ class ThermalMonitor(ProcessTaskBase):
 class ThermalControlDaemon(daemon_base.DaemonBase):
     # Interval to run thermal control logic
     INTERVAL = 60
+    RUN_POLICY_ELAPSE_THRESHOLD = 30
```
Comment (jleveque) [inline sonic-thermalctld/scripts/thermalctld:610]: This constant could be more descriptive and also include units, e.g., `RUN_POLICY_WARN_THRESHOLD_SECS`

Technically speaking, the other interval-related constants could use to have units added, also, but this can be saved for a separate PR.
